phonebuddies/DatabaseConnector.py

```diff
@@ -111,7 +111,11 @@ def get_all_buddies_map(self):
         buddy_email_map = {}
 
         for buddy in self.get_all_buddies():
-            buddy_email_map[buddy.email] = buddy
+            buddy_email_map[buddy.email.lower()] = buddy  # Converts buddy.email to lower before adding to map
```

Comment:
Do you need to do this given it should be `lower`ed when ingested from the database?